src/SlackConnector/SlackConnection.cs

```diff
@@ -223,6 +249,27 @@ public void Disconnect()
                 Type = SlackChatHubType.DM
             };
         }
+        public async Task CloseDirectMessageChannel(string channel)
```

Comment:
Can you add tests against these new methods. Also, spacing issue with method above